databases/week1/exercise.sql

```diff
@@ -0,0 +1,42 @@
+USE hyf_lesson1;
+
+-- Select the names and phones of all users
+Select name, phone FROM hyf_lesson1.user;
```

Comment:
please have a look at the homework section at this page https://github.com/HackYourFuture-CPH/databases/tree/main/lesson1